zerver/views/auth.py

```diff
@@ -729,7 +730,7 @@ def login_page(
     request: HttpRequest,
     next: str = REQ(default="/"),
     **kwargs: Any,
-) -> HttpResponse:
+) -> HttpResponseBase:
```

Comment:
Same question: why do these auth views return `HttpResponseBase`?

Reply:
These auth views get the response from the `.as_view` function, which returns a response of the type `HttpResponseBase`.